multicluster/test/e2e/main_test.go

```diff
@@ -104,4 +104,7 @@ func TestConnectivity(t *testing.T) {
 		testMCAntreaPolicy(t, data)
 		tearDownForPolicyTest()
 	})
+	// Wait 5 seconds to let both member and leader controllers to clean up all resources
+	// otherwise, Namespace deletion may stuck into termininating status.
```

Comment:
why did it stuck?

Reply:
Leader controller is using Finalizer for ResourceExport, which means only when corresponding ResourceImport is deleted,  ResourceExport will be truly deleted. If we exit test and clean up Namespace directly, it will stuck.

Reply:
But what does the sleep block if it's added here? `teardownTest` does nothing but clean up the log dir. Which namespace you mean here? test or kube-system?

Reply:
The Namespace is deleted in test-mc.sh here https://github.com/antrea-io/antrea/blob/20663c2727943f284eef44273556a1331897ffcd/ci/jenkins/test-mc.sh#L144

Reply:
Got it. I would not block this PR due to this, but I would suggest moving such cleanup confirmation to the test that creates it. It's worth to check if the resource is cleaned up as expected in time when components are running well, and could reduce the overall time.

Reply:
Sure, thanks.